getnf

```diff
@@ -48,6 +55,28 @@ for i in $listFonts; do
 		if [ -z "$checkFont" ]; then #If the font already is installed, skip it
 				echo "$i font download started" &&
 						#download the font
+        if [[ "$os" == 'Darwin' ]]; then
+          (cd $downDir; curl -LJO -\# "https://github.com/ryanoasis/nerd-fonts/releases/download/$release/$i.zip" \
+            -o $i.zip &&
+						echo "$i font download finished" &&
+						echo "$i font unziping started" &&
+						#Unzipe the downloaded archive
+						unzip -qq $downDir/$i.zip -d $distDir &&
+						echo "$i font unzipping finished" &&
+						echo "Font $i Installed" &&
+						echo "$i provides:" &&
+						installedFontName=$(curl --silent "$nerdfontsrepo/contents/patched-fonts/$i/Regular/complete?ref=master" | \
+									grep ".ttf" | \
+									awk -F ":" 'FNR == 1 {print $2}' | \
+									awk '{print $1}' | \
+								        sed 's/"//g') &&
+						fc-list | grep -i $installedFontName | \
+						    awk -F "/" '{print $7}' | \
+						    sed 's/style\=//' |  \
+						    awk -F ":" 'BEGIN {print "FONT NAME" " | " "FILE NAME" " | " "STYLE"} {print $2 " | " $1 " | " $3}' | \
+						#set this variable to true so that the font cache get's updated
+            aFontInstalled=True)
+        else
```

Comment:
all of this is redundant as it is doing the same thing as in the else block, except the curl part, which I recommend not to do with a `cd` and just let curl output the file to the output directory which is set as the downDir

Reply:
Output dir is not supported on mac

Reply:
Updated